test/test-tlc-green-taxi-trip.rb

```diff
@@ -0,0 +1,59 @@
+class TLCGreenTaxiTripTest < Test::Unit::TestCase
+  def record(*args)
+    DatasetsParquet::TLC::GreenTaxiTrip::Record.new(*args)
+  end
+
+  test("each") do
+    dataset = DatasetsParquet::TLC::GreenTaxiTrip.new(year: 2022, month: 1)
+    records = dataset.each.to_a
+
+    assert_equal([
+                   62495,
+                   record(2,
+                          Time.parse('2022-01-01 09:14:21 +0900'),
+                          Time.parse('2022-01-01 09:15:33 +0900'),
+                          "N",
+                          1.0,
+                          42,
+                          42,
+                          1.0,
+                          0.44,
+                          3.5,
+                          0.5,
+                          0.5,
+                          0.0,
+                          0.0,
+                          nil,
+                          0.3,
+                          4.8,
+                          2.0,
+                          1.0,
+                          0.0).to_h,
```

Comment:
Could you use a `Hash` literal instead of `record().to_h` to confirm that we produce an expected `Hash`?

Reply:
fix: [e6c8bdc](https://github.com/red-data-tools/red-datasets-parquet/pull/4/commits/e6c8bdc24510807dc52af96cab795880404cc1a3) Thanks, this test case gets much more clear than before.